packages/documentation/src/pages/architecture/unit-tests.mdx

```diff
@@ -0,0 +1,120 @@
+## How do unit tests work?
+
+#### Where they live
```

Comment:
>Location 

The content switches between "live" and "located"- might seem dry but it's easier to scan when documentation uses one word for a thing.